### DISCUSSION TITLE
getting more granural control of the fans

### DISCUSSION THREAD

**Discussion by russellmm** [BartoszCichecki/LenovoLegionToolkit#959] [Ideas]
The Fan control of my new Legion  Slim 7 2023 is simply terrible... I have been searching for solution and while I immediately loaded your software the day I got the laptop, I see you are forced to mimic Legion implementation.

Is getting control of more data points and/or adjusting the temps of the data points simply not possible? 

I am curious how this Linux repo is doing it (maybe older Legions only?):
https://github.com/johnfanv2/LenovoLegionLinux/tree/main

P.S. Legionfancontrol does not seem to support the new 2023 Slim 7 Legion at this point.

**Comment by BartoszCichecki** [comment]
Already asked many times. It's not possible.
